### DISCUSSION TITLE
Think I have a misunderstanding trying to test an external spec

### DISCUSSION THREAD

**Discussion by jameswagoner** [hotmeteor/spectator#193] [Q&A]
I would like to test against Twilio's open api spec. Least that is what I think I want to do.

Currently I am stuck getting 404s for any type of request I make and the messages looks like its complaining that a laravel route cannot be found. Is that what I am missing in all this? You must actually have routes in your laravel app when using `Spectator::using()`?

If that is the case, is what I want to do possible? I don't want to actually send a text message in my tests, just want to know if what payload is valid.

**Comment by bastien-phi** [comment]
@jameswagoner Interesting use-case. As far as I understand, you would like to test if your HTTP call like
```php
Http::post('https://twilio.url/api/route', $params);
```
is correct accordingly to the openapi, right ?

Actually, this is not possible with Spectator. Its goal is to verify if *your* Laravel API respects the openapi definition.
